src/mswin/fg_window_mswin.c

```diff
@@ -661,35 +661,30 @@ void fgPlatformOpenWindow( SFG_Window* window, const char* title,
     }
 #endif /* !defined(_WIN32_WCE) */
 
+    TCHAR* tstr = fghTstrFromStr(title);
```

Comment:
For maximum compatibility, freeglut is written in C89. You can't define variables everywhere in the code, only at the top of a scope. Please move the delcaration of tstr to the start of the function.